src/Appium.Net/Appium/Android/AndroidCommandExecutionHelper.cs

```diff
@@ -117,6 +117,9 @@ public static ConnectionType GetConection(IExecuteMethod executeMethod)
                 PrepareArguments(new[] {"phoneNumber", "action"},
                     new object[] {number, gsmCallAction.ToString().ToLowerInvariant()}));
 
+        public static void SetGsmStrength(IExecuteMethod executeMethod, GsmSignalStrength gsmSignalStrength) =>
+            executeMethod.Execute(AppiumDriverCommand.SetGsmSignalStrength, PrepareArgument("signalStrength", gsmSignalStrength));
```

Comment:
is the enum value automatically converted to string when passed to the request body?

Reply:
From the top of my memory while testing, it passes the enum index value

Reply:
I'l double check now

Reply:
Yeah it passes the enum value, like so:
`{"signalStrength": 2}`
Prefer the string?

Reply:
I prefer it to be more explicit, but not sure how explicit this is for C# in particular. Leaving it up to you

Reply:
I'm happy with the enum value